charts/castai-agent/values.yaml

```diff
@@ -41,6 +41,24 @@ resources: {}
 additionalEnv:
   PPROF_PORT: "6060"
 
+additionalEnvFromRef:
```

Comment:
Could we name this variable `extraEnvs` or `additionalEnvs`, something along those lines. Because it supports the usual `name: FOO value: BAR` env objects, not just `valueFrom`.